### DISCUSSION TITLE
MSCS%40UVa/

### DISCUSSION THREAD

**Discussion by giscus** [opencsapp/opencsapp.github.io#218] [Announcements]
# MSCS%40UVa/

None

https://opencs.app/MSCS%40UVa/

**Comment by yuhengcai1** [comment]
推荐上调，这个项目不至于B-

**Reply by Geeksongs** [reply]
这个项目应该和rice cs难度差不多？a-档左右

**Reply by Geeksongs** [reply]
我看有不少rice录取的，但uva mscs被拒了。但也有被uva mscs录取，被rice cs拒了。差不多对半开，感觉难度差不多

**Comment by sikong206573** [comment]
国内申请这个学校的人少是因为佛吉尼亚是以前南方棉花庄园所在地，白人至上主义比较盛行吗

**Comment by Perrkkey** [comment]
双非录取要求90+，就算上了录取概率也很小
